data-prepper-plugins/kafka-plugins/src/main/java/org/opensearch/dataprepper/plugins/kafka/source/KafkaSource.java

```diff
@@ -118,7 +118,7 @@ public KafkaSource(final KafkaSourceConfig sourceConfig,
     @Override
     public void start(Buffer<Record<Event>> buffer) {
         Properties authProperties = new Properties();
-        KafkaSourceSecurityConfigurer.setAuthProperties(authProperties, sourceConfig, LOG);
+        KafkaConsumerSecurityConfigurer.setAuthProperties(authProperties, sourceConfig, LOG);
```

Comment:
Is this (security configurer) applicable only to Consumer? I think it would be applicable to producer as well.

Reply:
Probably rename it to just "KkafkaSecurityConfigurer" and remove "Source' or "Consumer" from the name.

Reply:
Done